packages/valaxy/package.json

```diff
@@ -82,7 +81,7 @@
     "critters": "^0.0.24",
     "cross-spawn": "^7.0.3",
     "css-i18n": "^0.0.2",
-    "date-fns": "^3.6.0",
```

Comment:
Since `date-fns-tz` already includes `date-fns` (as [package.json](https://github.com/marnusw/date-fns-tz/blob/00b48b2cd4505a69203aac5734773114bce13204/package.json#L131C1-L133C5)), I believe there's no need for an additional installation. It works correctly in the current environment. I think I will not use the `date-fns` library directly to avoid unnecessary dependencies.

Reply:
This is `peerDependencies`, will not be installed with `date-fns-tz`.
We should install it as dependencies of valaxy.

Reply:
Starting with npm v7, the automatic installation of peerDependencies has been reintroduced. This setting will be enabled by default [auto-install-peers](https://pnpm.io/npmrc#auto-install-peers) from pnpm v8. I just want to make it better, and if you don't like it, you can change it anytime.